src/event-handlers/tcr-emails.js

```diff
@@ -0,0 +1,83 @@
+const _web3 = require('../utils/web3')
+const _sendgrid = require('../utils/sendgrid')
+const _t2cr = require('../assets/contracts/ArbitrableTokenList.json')
+const dynamoDB = require('../utils/dynamo-db')
+
+const handlers = {
+  Dispute: async (_, t2cr, event) => {
```

Comment:
Unnecessary first param.

Reply:
There doesn't seem to be a consensus on how to denote unused parameters, so I use underscore to be consistent (as in https://github.com/kleros/event-microservices/blob/2169e36fc8a9336396ea5fb0a5c7649eca6ca125/src/event-handlers/kleros-liquid-emails.js#L7).

Reply:
Yeah, but why do you have that parameter there at all? In Kleros Liquid it's used for `TokenAndETHShift`. You never need it here.

Reply:
I see what you mean. Will remove it.